### PR TITLE
fix test certificate for KKP installer

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
Suddenly the cert-manager started to reject my cert.

**Does this PR introduce a user-facing change?**:
```release-note
Fix installer trying an invalid certificate to test cert-manager
```

### CODE REVIEW

File: pkg/install/stack/kubermatic-master/certmanager.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/sirupsen/logrus"
 
 	certmanagerv1alpha2 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha2"
+	v1 "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
```
Comment (moadqassem) [inline pkg/install/stack/kubermatic-master/certmanager.go:27]: rearrange groups ;)

Comment (xrstf) [reply pkg/install/stack/kubermatic-master/certmanager.go:27]: Fixed

**Review by moadqassem — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**